src/utils/mapHtmlAttributesToReactElementAttributes.ts

```diff
@@ -32,21 +32,25 @@ const getParsedAttributeValue = function (attribute: string, value: string) {
 export function mapHtmlAttributesToReactElementAttributes(
   attributes: Record<string, string>
 ): Record<string, string> {
-  return Object.keys(attributes)
-    .filter((attr) => isValidTagOrAttributeName(attr))
-    .reduce((mappedAttributes, attribute) => {
```

Comment:
I don't see any benefit or reason in replacing the reduce with a for-loop except personal preference for loops.
Please revert this back to reduce.